tests/baselines/reference/ambientShorthandFindAllRefs.baseline.jsonc

```diff
@@ -1,6 +1,74 @@
 // === /tests/cases/fourslash/user.ts ===
 // import {/*FIND ALL REFS*/[|x|]} from "jquery";
 
+// === 0 ===
+  // === Definition ===
+    // containerKind: ""
+    // containerName: ""
+    // kind: "alias"
+    // name: "(alias) module \"jquery\"\nimport x"
+    // displayParts: [
```

Comment:
One thing I'm grappling with here is that it seems like all of this info here is a duplicate of the information that's already in these baslines beforehand; is it possible to only add the new info instead of duplicating the display parts and such instead? It seems like that's a lot of the source of the huge amount of added lines.

Reply:
It’s Some of the info that tests were checking as part of range additional data .. now I feel like it’s one blob if it’s json and so hard to decipher so I added it here to make easier to read .. contemplated removing json part all together but we would miss additional data properties when added so I let it be .. not sure what’s better approach here

Reply:
Eg there are some tests that check display parts , most check write acces in references etc